scripts/emoji_codegen.py

```diff
@@ -68,16 +105,67 @@ def generate_qml_list(**kwargs):
         code, qualification, charAndName = segments
 
         # skip unqualified versions of same unicode
-        if qualification == 'unqualified':
-            continue
-
-        if qualification == 'component':
+        if qualification != 'fully-qualified' and qualification != 'component' :
             continue
+        
 
         char, name = re.match(r'^(\S+) E\d+\.\d+ (.*)$', charAndName).groups()
-
-        categories[current_category].append(Emoji(code, name))
+        shortname = name
+        
+        # discard skin tone variants for sanity
+        # __contains__ is so stupid i hate prototype languages
+        if name.__contains__("skin tone") and qualification != 'component': 
+            continue
+        if qualification == 'component' and not name.__contains__("skin tone"): 
+            continue
+        #TODO: Handle skintone modifiers in a sane way
```

Comment:
Until we handle skin tones differently, can we keep them in the picker? It would be annoying to not be able to send those variants anymore...